modules/services/model/src/main/java/au/com/shiftyjelly/pocketcasts/models/db/dao/PlaylistDao.kt

```diff
@@ -98,6 +105,21 @@ abstract class PlaylistDao {
         }
     }
 
+    @Query(
+        """
+        SELECT
+          COUNT(*) AS episode_count,
+          SUM(MAX(0, IFNULL(podcastEpisode.duration, 0) - IFNULL(podcastEpisode.duration, 0))) AS time_left
```

Comment:
```suggestion
          SUM(MAX(0, IFNULL(podcastEpisode.duration, 0) - IFNULL(podcastEpisode.played_up_to, 0))) AS time_left
```
Should this use `played_up_to` to figure out the time left?